### PR TITLE
Support SOPS age encryption

### PR DESCRIPTION
**Pull request by hiddeco**
Fixes: #306

### CODE REVIEW

File: controllers/kustomization_decryptor.go [hunk 1]
```diff
@@ -136,18 +138,25 @@ func (kd *KustomizeDecryptor) ImportKeys(ctx context.Context) error {
...
+				if err != nil {
+					return err
+				}
+				if err := ioutil.WriteFile(keyPath, file, os.ModePerm); err != nil {
+					return fmt.Errorf("unable to write key to storage: %w", err)
+				}
+				if err := kd.gpgImport(keyPath); err != nil {
+					return err
+				}
+			case ".txt":
```
Comment (hiddeco) [inline controllers/kustomization_decryptor.go:155]: Wondering if this very generic extension is going to bite us later if some other encryption format is introduced, but do not have a clue about good alternatives, given `.age` is reserved for encrypted age files (and not keys).

Comment (stefanprodan) [reply controllers/kustomization_decryptor.go:155]: What about `.agekey`? the Age keygen doesn't have opinions about the file extension, but SOPS does... as `txt` is hoardcoded.

Comment (hiddeco) [reply controllers/kustomization_decryptor.go]: Think `.agekey` is an OK suggestion, SOPS guards itself against collisions by looking for a `keys.txt` file in a specific (age) folder.

**Review by stefanprodan — APPROVED** [review]
LGTM

Awesome work @hiddeco 🏅

**Pull request merged**